project/Build.scala

```diff
@@ -33,15 +33,14 @@ object MinimalBuild extends Build {
     resolvers += typesafeSnapshot,
     resolvers += sonatypeSnapshot,
     // publishArtifact in packageDoc := false,
-    crossScalaVersions := Seq("2.10.4","2.11.5"),
-    scalaVersion := "2.11.5",
+    scalaVersion := "2.11.7",
```

Comment:
Although it doesn't seem to make any effect, any specific reason for this version bump?